src/main/java/frc/team2412/robot/commands/intake/IntakeInCommand.java

```diff
@@ -21,7 +21,8 @@ public void execute() {
 
     @Override
     public boolean isFinished() {
-        return true;
+        return subsystem.hasOpposingColorCargo();
```

Comment:
? This is identical to IntakeInIdentifyBallCommand. I'd recommend having this return 'false' as the command never really finishes (the intake keeps spinning) and it'd be useful to have a command that intakes balls regardless of color.